tools/tests.py

```diff
@@ -24,13 +24,14 @@
 CORTEX_ARM_SUPPORT = {}
 
 for target in TARGETS:
-    DEFAULT_SUPPORT[target.name] = target.supported_toolchains
-
-    if target.core.startswith('Cortex'):
-        CORTEX_ARM_SUPPORT[target.name] = [
-            t for t in target.supported_toolchains
-            if (t == 'ARM' or t == 'uARM')
-        ]
+    if not target.is_TFM_target:
```

Comment:
Where are we documenting that tests.py can't run on TF-M targets? Do we ever want tests.py to run on TF-M targets in the future? If so, what would it take to make it work?

Reply:
I've created a JIRA task https://jira.arm.com/browse/IOTSEC-1265 to keep track of this requirement.